sopel/trigger.py

```diff
@@ -328,8 +333,8 @@ class Trigger(str):
     .. important::
 
         Use this attribute instead of the ``intent`` tag in :attr:`tags`.
-        Message intents never made it past the IRCv3 draft stage, and Sopel will
-        drop support for them in a future release.
+        Message intents never made it past the IRCv3 draft stage, and Sopel
+        dropped support for them in Sopel 8.
     """
```

Comment:
I'm gonna nitpick again, just like I did on your other PR today.

```suggestion

    """
```